### PR TITLE
run Symfony tests through Docker and Composer :whale: 

### PR DESCRIPTION
**Pull request by alexislefebvre**
Doc that go with https://github.com/symfony/symfony/pull/47102

### CODE REVIEW

File: contributing/code/tests.rst [hunk 1]
```diff
@@ -59,6 +62,62 @@ what's going on and if the tests are broken because of the new code.
 
         $ php ./phpunit src/Symfony/Component/Finder/
 
+Through Docker
+--------------
+
+You must `install Composer`_ and ``docker``.
+
+.. tip::
```
Comment (94noni) [inline contributing/code/tests.rst]: what about note instead of tip?

File: contributing/code/tests.rst [hunk 2]
```diff
@@ -59,6 +62,62 @@ what's going on and if the tests are broken because of the new code.
 
         $ php ./phpunit src/Symfony/Component/Finder/
 
+Through Docker
+--------------
+
+You must `install Composer`_ and ``docker``.
```
Comment (94noni) [inline contributing/code/tests.rst:72]: I will add something like:
Starting from v6.2, you can execute phpunit symfony tests inside docker with the folowing steps

Comment (alexisLefebvre) [reply contributing/code/tests.rst:72]: The PR targets 6.2 so it won't appear on previous versions.

Comment (alexisLefebvre) [reply contributing/code/tests.rst:72]: Ohhh, you meant an information box! I'll update this, thanks for the suggestion.

**Comment by alexislefebvre** [comment]
Let's close this, this feature didn't get much traction.

**Pull request closed without merging**